projects/elonkit/src/ui/audio-player/components/audio-player-time-slider/audio-player-time-slider.component.ts

```diff
@@ -0,0 +1,192 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  EventEmitter,
+  HostListener,
+  Input,
+  Output,
+  Renderer2,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import { MatSlider } from '@angular/material/slider';
+
+const TIME_LINE_RANGE = 8;
+const TOOLTIP_OFFSET = 21;
+
+@Component({
+  selector: 'es-audio-player-time-slider',
+  templateUrl: './audio-player-time-slider.component.html',
+  styleUrls: ['./audio-player-time-slider.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ESAudioPlayerTimeSliderComponent {
+  /** The duration in seconds of the current media resource. */
+  @Input() public duration: number;
+
+  /** The current playback position, in seconds */
+  @Input() public set audioCurrentTime(time: number) {
+    if (!this.elementTimeLine._isSliding) {
+      this._audioCurrentTime = time;
+    }
+  }
+  public get audioCurrentTime(): number {
+    return this._audioCurrentTime;
+  }
+  private _audioCurrentTime: number;
+
+  /** Emits the current playback position, in seconds. */
```

Comment:
Event emitted when current playback position is changed.
`new EventEmitter<number>();`